projects/adf-office-services-ext/src/lib/evaluators.ts

```diff
@@ -26,7 +26,15 @@
 import { RuleContext } from '@alfresco/adf-extensions';
 import { getFileExtension, supportedExtensions } from './utils';
 
-export function canOpenWithOffice(context: RuleContext): boolean {
+export interface AcaRuleContext extends RuleContext {
```

Comment:
Same here we are mixing the general acaRules with aosPlugin

Reply:
@DenysVuika can you check if this is correct?

Reply:
looks okay, we used this kind of interfaces before just to use Typescript type checking better for the evaluators/rules... in the current case in ACA the App Extensions Service is the RuleContext at the same time, so in order for a rule to access it's properties we use interfaces... without the interface, it would probably be `any`

```
export function canOpenWithOffice(context: any): boolean {
  if (!context.aosPlugin) {
    return false;
  }
```